pex/tools/commands/virtualenv.py

```diff
@@ -91,7 +91,7 @@ def create(
             interpreter.execute(args=["-m", "venv", "--without-pip", venv_dir])
         else:
             virtualenv_py = resource_string(__name__, "virtualenv_16.7.10_py")
-            with named_temporary_file(mode="w") as fp:
+            with named_temporary_file(mode="wb") as fp:
```

Comment:
Is the `bytes`/`str`-ness of `resource_string` version dependent?

Reply:
It doesn't look like it - at least as far back as 38.2.0 which was my lazy check. I'm forgetting now why I hit an error here in this PR and not when the class was introduced.